packages/manfred2md/src/engine/md-parts/studies-section/studies-section.ejs

```diff
@@ -17,16 +17,16 @@
 - **Grado obtenido:** <%= study.degreeAchieved ? "No obtenido" : "Obtenido"%>
     <%_ } -%>
 
-    <%_ if ( study?.startDate && study?.finishDate ) { -%>
+    <%_ if ( study?.finishDate ) { -%>
```

Comment:
FinishDate is optional but startDate is required. If finishDate is null or undefined the mapper alwais return a empty string. If finishDate === "", what happen here?.... remove this condition, please.

Reply:
Thank you Juan Pablo 😃 , it has become clear to me, I just uploaded the change 👌🏻